src/main/java/com/minecolonies/core/entity/ai/workers/AbstractEntityAIStructureWithWorkOrder.java

```diff
@@ -219,14 +219,24 @@ private void requestMaterialsState()
     @Override
     protected IAIState waitForRequests()
```

Comment:
not sure if the override is still needed at all, since its on load the AI never starts in this state